Resources/Prototypes/_Goobstation/GameRules/midround.yml

```
@@ -51,7 +51,7 @@
     tags:
       - MidroundAntag
 
-# Midround Changelings
+ Midround Changelings
```

This will cause test fails. Why did you uncomment this? This was a comment for the section identifier, a header so-to-speak.
```suggestion
 # Midround Changelings
```